blueprints/node-webkit/index.js

```diff
@@ -20,12 +20,13 @@ module.exports = {
     var _this = this;
     var dependencies = this.project.dependencies();
 
-    return this.addNwConfig(options)
-      .then(function() {
-        if (!dependencies.nw) {
-          return _this.addPackageToProject('nw');
-        }
-      });
+    return this.addNwConfig(options).then(function () {
```

Comment:
I couldn't specify this as a JSCS rule, but when using promises, especially with multiple `then`'s, start each `.then()` on a new line and indent, similar to this [example](https://github.com/ember-cli/ember-cli/blob/v0.2.3/lib/commands/init.js#L75-L89).